src/docfx/template/TemplateEngine.cs

```diff
@@ -74,7 +74,8 @@ public static bool IsMigratedFromMarkdown(string? mime)
         {
             return "Hub".Equals(mime, StringComparison.OrdinalIgnoreCase) ||
                    "Landing".Equals(mime, StringComparison.OrdinalIgnoreCase) ||
-                   "LandingData".Equals(mime, StringComparison.OrdinalIgnoreCase);
+                   "LandingData".Equals(mime, StringComparison.OrdinalIgnoreCase) ||
+                   "Architecture".Equals(mime, StringComparison.OrdinalIgnoreCase);
```

Comment:
since this list grows, how about creating a hashset for it and use `Contains` to check instead?